### PR TITLE
Fix rolling build

### PR DESCRIPTION
**Pull request by vatanaksoytezer**
These two print statements are breaking the rolling build.

### CODE REVIEW

**Comment by matthew-reynolds** [comment]
Let's stick with printf-style rather than stringstream style (See #175), if nothing else for consistency in the codebase.

**Comment by vatanaksoytezer** [comment]
Sure, I'll change them to printf style

**Comment by vatanaksoytezer** [comment]
Ping to @bmagyar to not forget this before a rolling release

**Review by destogl — APPROVED** [review]

File: joint_trajectory_controller/src/joint_trajectory_controller.cpp [hunk 1]
```diff
@@ -340,7 +340,7 @@ JointTrajectoryController::on_configure(const rclcpp_lifecycle::State &)
     auto it = std::find(
       allowed_interface_types_.begin(), allowed_interface_types_.end(), interface);
     if (it == allowed_interface_types_.end()) {
-      RCLCPP_ERROR(logger, "Command interface type '" + interface + "' not allowed!");
+      RCLCPP_ERROR(logger, "Command interface type %s not allowed!", interface.c_str());
```
Comment (bmagyar) [inline joint_trajectory_controller/src/joint_trajectory_controller.cpp:343]:
```suggestion
      RCLCPP_ERROR(logger, "Command interface type '%s' not allowed!", interface.c_str());
```

File: joint_trajectory_controller/src/joint_trajectory_controller.cpp [hunk 2]
```diff
@@ -421,7 +421,7 @@ JointTrajectoryController::on_configure(const rclcpp_lifecycle::State &)
     auto it = std::find(
       allowed_interface_types_.begin(), allowed_interface_types_.end(), interface);
     if (it == allowed_interface_types_.end()) {
-      RCLCPP_ERROR(logger, "State interface type '" + interface + "' not allowed!");
+      RCLCPP_ERROR(logger, "State interface type %s not allowed!", interface.c_str());
```
Comment (bmagyar) [inline joint_trajectory_controller/src/joint_trajectory_controller.cpp:424]:
```suggestion
      RCLCPP_ERROR(logger, "State interface type '%s' not allowed!", interface.c_str());
```

**Review by bmagyar — APPROVED** [review]

**Pull request merged**